tools/check_format_test_run_under_docker.py

```diff
@@ -105,11 +111,17 @@ def checkFileExpectingError(filename, expected_substring):
 def checkFileExpectingOK(filename):
   command, status, stdout = runCheckFormat("check", getInputFile(filename))
   if status != 0:
-    print("status=%d, output:\n" % status)
+    logMessage(1, "status=%d, output:\n" % status)
```

Comment:
Not a huge fan of magic numbers, so if we don't use `logging`, suggest we use some constants (e.g. `VERBOSE`, `INFO`, etc.).

Reply:
Using logging package now, so no longer an issue.